### PR TITLE
PAN-OS resource check for rules with no desc

### PR DESCRIPTION
**Pull request by jamesholland-uk**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

A resource check for PAN-OS Terraform provider for security policy rules. This one ensures that the description attribute is populated. This attribute can be set in two different resource types, both are covered by this check.

### CODE REVIEW

File: checkov/terraform/checks/resource/panos/PolicyDescription.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            # Iterate over each rule
+            for secrule in rules:
+
+                # Check if a description is defined in the rule
+                if 'description' in secrule:
+
+                    # If a description is defined, get the value
+                    desc = secrule.get('description')
+
+                    if desc[0] == "":
```
Comment (gruebel) [inline checkov/terraform/checks/resource/panos/PolicyDescription.py:33]:
```suggestion
                    if desc[0].strip() == "":
```
how about adding `strip` here, then we can even trick the sneaky ones, who just add a space or multiple 😄

**Review by gruebel — COMMENTED** [review]
nice, looks good, let me know, what you think about my comment, otherwise it is good to merge 🚀

File: checkov/terraform/checks/resource/panos/PolicyDescription.py (hunk 1 above)
Comment (jamesholland-uk) [reply checkov/terraform/checks/resource/panos/PolicyDescription.py:33]: Yes, I like your suggestion 👍

**Comment by jamesholland-uk** [comment]
⬆️ And some tests for the strings composed of spaces too
Good to go if you're happy Anton?

**Review by gruebel — APPROVED** [review]
great thanks 🏆

**Pull request merged**